azure-pipelines.yml

```diff
@@ -246,9 +246,11 @@ jobs:
     condition: eq(variables['nightly_check'], 'true')
     displayName: 'Change name to giotto-tda-nightly'
 
+  # Set BOOST_ROOT_PIPELINE to the version used in the pipeline
+  # See https://github.com/actions/virtual-environments/issues/687#issuecomment-616345933
   - script: |
-      python .azure-ci/install_boost.py || exit /b
-    displayName: 'Install boost'
+      echo "##vso[task.setvariable variable=BOOST_ROOT_PIPELINE]%BOOST_ROOT_1_72_0%"
```

Comment:
In Azure, the only printout to the console seems to be https://dev.azure.com/maintainers/Giotto/_build/results?buildId=2036&view=logs&j=9c643c56-8d93-5243-a62b-ae3500fd2753&t=37987b60-a129-55b7-a785-02333769801c&l=11. Is this what you expect?

Reply:
Yes, it's not a print to the output itself, but it's a command you send, see [docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#set-variables-in-scripts). This allow me to set a variable from a environment variable. 

I tried different solutions, but it's the only one I made it work.
I would prefer a solution like [this](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#set-variables-in-pipeline), but I could not read from a environment variable ... If you have a better solution, let me know. I'm not at all an expert on this topic.

Reply:
Thanks for the reply.  I guess I was expecting the presence of `echo` to mean I would get some prints which at least look related to the command sent. But I have no idea what
```
"C:\windows\system32\cmd.exe" /D /E:ON /V:OFF /S /C "CALL "D:\a\_temp\246965b8-4c61-438d-8e9a-10126f523db0.cmd""
```
has to do with the command sent.

Reply:
Well, it seems more related on the pipeline when we call a `-script` line.
If you look also on different steps, we can see a similar output at the beginning:

* [here](https://dev.azure.com/maintainers/Giotto/_build/results?buildId=2036&view=logs&j=9c643c56-8d93-5243-a62b-ae3500fd2753&t=d086efcb-3547-5b32-71d0-a0488cde9cd4)
* [here](https://dev.azure.com/maintainers/Giotto/_build/results?buildId=2036&view=logs&j=9c643c56-8d93-5243-a62b-ae3500fd2753&t=b18e1b75-0cfd-5a82-ac92-0f6e32edf036)
* etc ...

I think it's more related to the command executed on the current step.